pkg/rpc/client/nns/record.go

```diff
@@ -0,0 +1,27 @@
+package nns
```

Comment:
So, can you reuse this package between the contract and client?

Reply:
I thought about that, but importing `.../pkg/rpc/client/nns` is weird to see in contract.

Reply:
I also for importing this code from the contract, we don't need duplication.

Reply:
Let's solve this in #1865.